app/Http/Controllers/BuslineController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Busline;
+use Illuminate\Http\Request;
+
+class BuslineController extends Controller
+{
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
```

Comment:
It's better to return some information about unimplemented feature than doing nothing. You can return 501 error code: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501
```
return response()->json([], Response::HTTP_NOT_IMPLEMENTED);
```